### PR TITLE
[zh] Resync deployment page

### PR DESCRIPTION
**Pull request by mengjiao-liu**
Fixes: #32316

Manually check for out-of-sync.

### CODE REVIEW

File: content/zh-cn/docs/concepts/workloads/controllers/deployment.md [hunk 1]
```diff
@@ -798,7 +798,7 @@ Deployment 被触发上线时，系统就会创建 Deployment 的新的修订版
     Labels:  app=nginx
     Containers:
      nginx:
-      Image:        nginx:1.91
+      Image:        nginx:1.161
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/workloads/controllers/deployment.md:801]: 1.16.1 ？

Comment (mengjiao-liu) [reply content/zh-cn/docs/concepts/workloads/controllers/deployment.md:801]: 这是为验证 deployment 回滚，故意设置的错误的 nginx tag。

Comment (tengqm) [reply content/zh-cn/docs/concepts/workloads/controllers/deployment.md:801]: OK，了解，没细看上下文。

**Review by ydFu — APPROVED** [review]
In the deployment page, the version issues related to `nginx` have been resynchronized with English.
Nice catch!

/lgtm

**Pull request merged**